src/components/githubRepos.js

```diff
@@ -0,0 +1,64 @@
+import React, {Component} from 'react';
+import Repo from './repo'
+
+
+// const styles = {
```

Comment:
No commented code on PRs unless you absolutely need it, in which case there should be a comment saying so.